> Build error
> 
> > D:\a_work\1\s\src\Controls\tests\TestCases.HostApp\Issues\Issue3262.cs(48,16): error CS0118: 'Windows' is a namespace but is used like a type [D:\a_work\1\s\src\Controls\tests\TestCases.HostApp\Controls.TestCases.HostApp.csproj::TargetFramework=net9.0-windows10.0.19041.0]
> > D:\a_work\1\s\src\Controls\tests\TestCases.HostApp\Issues\Issue3262.cs(48,5): error CS1929: 'IPlatformElementConfiguration<Windows, WebView>' does not contain a definition for 'SetIsJavaScriptAlertEnabled' and the best extension method overload 'WebView.SetIsJavaScriptAlertEnabled(IPlatformElementConfiguration<Windows, WebView>, bool)' requires a receiver of type 'Microsoft.Maui.Controls.IPlatformElementConfiguration<Microsoft.Maui.Controls.PlatformConfiguration.Windows, Microsoft.Maui.Controls.WebView>' [D:\a_work\1\s\src\Controls\tests\TestCases.HostApp\Controls.TestCases.HostApp.csproj::TargetFramework=net9.0-windows10.0.19041.0]
> > D:\a_work\1\s\src\Controls\tests\TestCases.HostApp\Issues\Issue3262.cs(48,16): error CS0118: 'Windows' is a namespace but is used like a type [D:\a_work\1\s\src\Controls\tests\TestCases.HostApp\Controls.TestCases.HostApp.csproj::TargetFramework=net9.0-windows10.0.20348.0]
> > D:\a_work\1\s\src\Controls\tests\TestCases.HostApp\Issues\Issue3262.cs(48,5): error CS1929: 'IPlatformElementConfiguration<Windows, WebView>' does not contain a definition for 'SetIsJavaScriptAlertEnabled' and the best extension method overload 'WebView.SetIsJavaScriptAlertEnabled(IPlatformElementConfiguration<Windows, WebView>, bool)' requires a receiver of type 'Microsoft.Maui.Controls.IPlatformElementConfiguration<Microsoft.Maui.Controls.PlatformConfiguration.Windows, Microsoft.Maui.Controls.WebView>' [D:\a_work\1\s\src\Controls\tests\TestCases.HostApp\Controls.TestCases.HostApp.csproj::TargetFramework=net9.0-windows10.0.20348.0]

Hmm not sure what the problem is. I added a commit that may fix it

--- a/src/Controls/tests/TestCases.HostApp/Issues/Issue3262.cs
+++ b/src/Controls/tests/TestCases.HostApp/Issues/Issue3262.cs
@@ -13,19 +13,16 @@ namespace Maui.Controls.Sample.Issues
 
 		protected override void Init()
 		{
-#pragma warning disable CS0612 // Type or member is obsolete
 			Label header = new Label
 			{
 				Text = "Cookies...",
-				FontSize = Device.GetNamedSize(NamedSize.Medium, typeof(Label)),
 				HorizontalOptions = LayoutOptions.Center
 			};
-#pragma warning restore CS0612 // Type or member is obsolete
 
 			try
 			{
 				CookieContainer cookieContainer = new CookieContainer();
-				string url = "https://dotnet.microsoft.com/apps/xamarin";
+				string url = "https://dotnet.microsoft.com/apps/maui";
 				Uri uri = new Uri(url, UriKind.RelativeOrAbsolute);
 
 				Cookie cookie = new Cookie
@@ -39,35 +36,39 @@ namespace Maui.Controls.Sample.Issues
 
 				cookieContainer.Add(uri, cookie);
 
-#pragma warning disable CS0618 // Type or member is obsolete
 				WebView webView = new WebView
 				{
 					Source = url,
-					HorizontalOptions = LayoutOptions.FillAndExpand,
-					VerticalOptions = LayoutOptions.FillAndExpand,
+					HeightRequest = 200,
+					WidthRequest = 300,
 					Cookies = cookieContainer
 				};
-#pragma warning restore CS0618 // Type or member is obsolete
-				webView.On<Microsoft.Maui.Controls.PlatformConfiguration.Windows>().SetIsJavaScriptAlertEnabled(true);
+
+				webView.On<Windows>().SetIsJavaScriptAlertEnabled(true);
 
 				Action<string> cookieExpectation = null;
 				var cookieResult = new Label()
 				{
 					Text = "Loading",
-					AutomationId = "CookieResult"
 				};
 
-				webView.Navigating += (_, __) =>
+				var successfullPageLoadLabel = new Label()
 				{
-					if (cookieExpectation != null)
-						cookieResult.Text = "Navigating";
+					IsVisible = false,
+					Text = "Page was loaded",
+					AutomationId = "SuccessfullPageLoadLabel"
+				};
+
+				var successCookiesLabel = new Label()
+				{
+					IsVisible = false,
+					Text = "Success",
+					AutomationId = "SuccessCookiesLabel"
 				};
 
 				webView.Navigated += async (_, __) =>
 				{
-					if (cookieResult.Text == "Loading")
-						cookieResult.Text = "Loaded";
-
+					successfullPageLoadLabel.IsVisible = true;
 					_currentCookieValue = await webView.EvaluateJavaScriptAsync("document.cookie");
 					cookieExpectation?.Invoke(_currentCookieValue);
 					cookieExpectation = null;
@@ -84,7 +85,12 @@ namespace Maui.Controls.Sample.Issues
 						{
 							Text = "Modify the Cookie Container"
 						},
-						cookieResult,
+						new HorizontalStackLayout()
+						{
+							cookieResult,
+							successfullPageLoadLabel,
+							successCookiesLabel
+						},
 						new StackLayout()
 						{
 							Orientation = StackOrientation.Horizontal,
@@ -97,7 +103,8 @@ namespace Maui.Controls.Sample.Issues
 									Command = new Command(() =>
 									{
 										webView.Cookies = cookieContainer;
-										cookieResult.Text = String.Empty;
+										cookieResult.Text = string.Empty;
+										successCookiesLabel.IsVisible = false;
 										cookieExpectation = (cookieValue) =>
 										{
 											if(cookieValue.Contains("TestCookie", StringComparison.OrdinalIgnoreCase))
@@ -106,7 +113,7 @@ namespace Maui.Controls.Sample.Issues
 											}
 											else
 											{
-												cookieResult.Text = "Success";
+												successCookiesLabel.IsVisible = true;
 											}
 										};
 
@@ -134,7 +141,7 @@ namespace Maui.Controls.Sample.Issues
 											}
 											else
 											{
-												cookieResult.Text = "Success";
+												successCookiesLabel.IsVisible = true;
 											}
 										};
 
@@ -149,6 +156,7 @@ namespace Maui.Controls.Sample.Issues
 									Command = new Command(() =>
 									{
 										cookieResult.Text = String.Empty;
+										successCookiesLabel.IsVisible = false;
 										cookieExpectation = (cookieValue) =>
 										{
 											if(Regex.Matches(cookieValue, "TestCookie").Count > 1)
@@ -157,7 +165,7 @@ namespace Maui.Controls.Sample.Issues
 											}
 											else
 											{
-												cookieResult.Text = "Success";
+												successCookiesLabel.IsVisible = true;
 											}
 										};
 
@@ -190,6 +198,7 @@ namespace Maui.Controls.Sample.Issues
 									{
 										webView.Cookies = cookieContainer;
 										cookieResult.Text = String.Empty;
+										successCookiesLabel.IsVisible = false;
 										cookieContainer.Add(new Cookie
 										{
 											Name = $"TestCookie{cookieContainer.Count}",
@@ -212,7 +221,7 @@ namespace Maui.Controls.Sample.Issues
 											}
 											else
 											{
-												cookieResult.Text = "Success";
+												successCookiesLabel.IsVisible = true;
 											}
 										};
 
@@ -242,6 +251,7 @@ namespace Maui.Controls.Sample.Issues
 										};
 
 										cookieResult.Text = String.Empty;
+										successCookiesLabel.IsVisible = false;
 										cookieExpectation = (cookieValue) =>
 										{
 											if(cookieValue.Contains(cookieToAdd.Name, StringComparison.OrdinalIgnoreCase))
@@ -250,7 +260,7 @@ namespace Maui.Controls.Sample.Issues
 											}
 											else
 											{
-												cookieResult.Text = "Success";
+												successCookiesLabel.IsVisible = true;
 											}
 										};
 
@@ -275,8 +285,13 @@ namespace Maui.Controls.Sample.Issues
 							AutomationId = "PageWithoutCookies",
 							Command = new Command(() =>
 							{
+								var previousCookies = webView.Cookies;
 								webView.Cookies = null;
 								webView.Source = "file:///android_asset/googlemapsearch.html";
+
+								//Restore to the previous state
+								webView.Cookies = previousCookies;
+								webView.Source = url;
 							})
 						}
 					}

--- a/src/Controls/tests/TestCases.HostApp/Issues/Issue3262.cs
+++ b/src/Controls/tests/TestCases.HostApp/Issues/Issue3262.cs
@@ -2,6 +2,8 @@
 using System.Text.RegularExpressions;
 using Label = Microsoft.Maui.Controls.Label;
 using WebView = Microsoft.Maui.Controls.WebView;
+using Microsoft.Maui.Controls.PlatformConfiguration;
+using Microsoft.Maui.Controls.PlatformConfiguration.WindowsSpecific;
 
 namespace Maui.Controls.Sample.Issues
 {

--- a/src/Controls/tests/TestCases.HostApp/Issues/Issue3262.cs
+++ b/src/Controls/tests/TestCases.HostApp/Issues/Issue3262.cs
@@ -1,5 +1,6 @@
 ﻿using System.Net;
 using System.Text.RegularExpressions;
+using Microsoft.Maui.Controls.PlatformConfiguration;
 using Microsoft.Maui.Controls.PlatformConfiguration.WindowsSpecific;
 using Label = Microsoft.Maui.Controls.Label;
 using WebView = Microsoft.Maui.Controls.WebView;

--- a/src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue3262.cs
+++ b/src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue3262.cs
@@ -14,7 +14,8 @@ namespace Microsoft.Maui.TestCases.Tests.Issues
 		public override string Issue => "Adding Cookies ability to a WebView...";
 
 		[Test]
-		[FailsOnMac]
+		[FailsOnMacWhenRunningOnXamarinUITest]
+		[FailsOnWindowsWhenRunningOnXamarinUITest]
 		public void LoadingPageWithoutCookiesSpecifiedDoesntCrash()
 		{
 			App.WaitForElement("SuccessfullPageLoadLabel");
@@ -24,8 +25,8 @@ namespace Microsoft.Maui.TestCases.Tests.Issues
 
 		[Test]
 		[Category(UITestCategories.Compatibility)]
-		[FailsOnMac]
-		[FailsOnWindows]
+		[FailsOnMacWhenRunningOnXamarinUITest]
+		[FailsOnWindowsWhenRunningOnXamarinUITest]
 		public void ChangeDuringNavigating()
 		{
 			App.WaitForElement("SuccessfullPageLoadLabel");
@@ -38,8 +39,8 @@ namespace Microsoft.Maui.TestCases.Tests.Issues
 
 		[Test]
 		[Category(UITestCategories.Compatibility)]
-		[FailsOnMac]
-		[FailsOnWindows]
+		[FailsOnMacWhenRunningOnXamarinUITest]
+		[FailsOnWindowsWhenRunningOnXamarinUITest]
 		public void AddAdditionalCookieToWebView()
 		{
 			App.WaitForElement("SuccessfullPageLoadLabel");
@@ -52,8 +53,8 @@ namespace Microsoft.Maui.TestCases.Tests.Issues
 
 		[Test]
 		[Category(UITestCategories.Compatibility)]
-		[FailsOnMac]
-		[FailsOnWindows]
+		[FailsOnMacWhenRunningOnXamarinUITest]
+		[FailsOnWindowsWhenRunningOnXamarinUITest]
 		public void SetToOneCookie()
 		{
 			App.WaitForElement("SuccessfullPageLoadLabel");
@@ -63,8 +64,8 @@ namespace Microsoft.Maui.TestCases.Tests.Issues
 
 		[Test]
 		[Category(UITestCategories.Compatibility)]
-		[FailsOnMac]
-		[FailsOnWindows]
+		[FailsOnMacWhenRunningOnXamarinUITest]
+		[FailsOnWindowsWhenRunningOnXamarinUITest]
 		public void SetCookieContainerToNullDisablesCookieManagement()
 		{
 			App.WaitForElement("SuccessfullPageLoadLabel");
@@ -77,8 +78,8 @@ namespace Microsoft.Maui.TestCases.Tests.Issues
 
 		[Test]
 		[Category(UITestCategories.Compatibility)]
-		[FailsOnMac]
-		[FailsOnWindows]
+		[FailsOnMacWhenRunningOnXamarinUITest]
+		[FailsOnWindowsWhenRunningOnXamarinUITest]
 		public void RemoveAllTheCookiesIAdded()
 		{
 			App.WaitForElement("SuccessfullPageLoadLabel");

--- a/src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue3262.cs
+++ b/src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue3262.cs
@@ -4,6 +4,7 @@ using UITest.Core;
 
 namespace Microsoft.Maui.TestCases.Tests.Issues
 {
+	[Category(UITestCategories.WebView)]
 	public class Issue3262 : _IssuesUITest
 	{
 		public Issue3262(TestDevice testDevice) : base(testDevice)
@@ -13,21 +14,21 @@ namespace Microsoft.Maui.TestCases.Tests.Issues
 		public override string Issue => "Adding Cookies ability to a WebView...";
 
 		[Test]
-		[Category(UITestCategories.WebView)]
-		[FailsOnAllPlatformsWhenRunningOnXamarinUITest]
+		[FailsOnMac]
 		public void LoadingPageWithoutCookiesSpecifiedDoesntCrash()
 		{
+			App.WaitForElement("SuccessfullPageLoadLabel");
 			App.Tap("PageWithoutCookies");
 			App.WaitForElement("PageWithoutCookies");
 		}
 
 		[Test]
-		[Category(UITestCategories.WebView)]
 		[Category(UITestCategories.Compatibility)]
-		[FailsOnAllPlatformsWhenRunningOnXamarinUITest]
+		[FailsOnMac]
+		[FailsOnWindows]
 		public void ChangeDuringNavigating()
 		{
-			App.WaitForElement("Loaded");
+			App.WaitForElement("SuccessfullPageLoadLabel");
 			// add a couple cookies
 			App.Tap("ChangeDuringNavigating");
 			ValidateSuccess();
@@ -36,12 +37,12 @@ namespace Microsoft.Maui.TestCases.Tests.Issues
 		}
 
 		[Test]
-		[Category(UITestCategories.WebView)]
 		[Category(UITestCategories.Compatibility)]
-		[FailsOnAllPlatformsWhenRunningOnXamarinUITest]
+		[FailsOnMac]
+		[FailsOnWindows]
 		public void AddAdditionalCookieToWebView()
 		{
-			App.WaitForElement("Loaded");
+			App.WaitForElement("SuccessfullPageLoadLabel");
 			// add a couple cookies
 			App.Tap("AdditionalCookie");
 			ValidateSuccess();
@@ -50,23 +51,23 @@ namespace Microsoft.Maui.TestCases.Tests.Issues
 		}
 
 		[Test]
-		[Category(UITestCategories.WebView)]
 		[Category(UITestCategories.Compatibility)]
-		[FailsOnAllPlatformsWhenRunningOnXamarinUITest]
+		[FailsOnMac]
+		[FailsOnWindows]
 		public void SetToOneCookie()
 		{
-			App.WaitForElement("Loaded");
+			App.WaitForElement("SuccessfullPageLoadLabel");
 			App.Tap("OneCookie");
 			ValidateSuccess();
 		}
 
 		[Test]
-		[Category(UITestCategories.WebView)]
 		[Category(UITestCategories.Compatibility)]
-		[FailsOnAllPlatformsWhenRunningOnXamarinUITest]
+		[FailsOnMac]
+		[FailsOnWindows]
 		public void SetCookieContainerToNullDisablesCookieManagement()
 		{
-			App.WaitForElement("Loaded");
+			App.WaitForElement("SuccessfullPageLoadLabel");
 			// add a cookie to verify said cookie remains
 			App.Tap("AdditionalCookie");
 			ValidateSuccess();
@@ -75,12 +76,12 @@ namespace Microsoft.Maui.TestCases.Tests.Issues
 		}
 
 		[Test]
-		[Category(UITestCategories.WebView)]
 		[Category(UITestCategories.Compatibility)]
-		[FailsOnAllPlatformsWhenRunningOnXamarinUITest]
+		[FailsOnMac]
+		[FailsOnWindows]
 		public void RemoveAllTheCookiesIAdded()
 		{
-			App.WaitForElement("Loaded");
+			App.WaitForElement("SuccessfullPageLoadLabel");
 			// add a cookie so you can remove a cookie
 			App.Tap("AdditionalCookie");
 			ValidateSuccess();
@@ -92,7 +93,7 @@ namespace Microsoft.Maui.TestCases.Tests.Issues
 		{
 			try
 			{
-				App.WaitForElement("Success");
+				App.WaitForElement("SuccessCookiesLabel");
 			}
 			catch
 			{

